torat_server/client_ishell.go

```diff
@@ -260,7 +270,9 @@ func (client *activeClient) Speedtest(c *ishell.Context) {
 
 	r := shared.Speedtest{}
 	if err := client.RPC.Call("API.Speedtest", void, &r); err != nil {
-		c.ProgressBar().Final(yellow("["+client.Client.Name+"] ") + red("[!] Could not perform speedtest on client:", err))
```

Comment:
Revert this line

Reply:
This is the same as a few other comments you have left. Again let's leave this for a patch where we fix all these. I would rather change everything at once.